packages/teleport/src/SideNav/SideNavExternalLink.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import defaultTheme from 'design/theme';
+import { color } from 'design/system';
+
+import { ArrowForward } from 'design/Icon';
+
+import SideNavItemIcon from './SideNavItemIcon';
+
+const fromTheme = ({ $nested = false, theme = defaultTheme }) => {
+  const css = {
+    position: 'relative',
+    fontSize: '12px',
+    fontWeight: theme.regular,
+    fontFamily: theme.font,
+    paddingLeft: theme.space[9] + 'px',
+    paddingRight: theme.space[3] + 'px',
+    background: theme.colors.primary.light,
+    color: theme.colors.text.secondary,
+    minHeight: '56px',
+
+    '&:active, &.active': {
+      borderLeftColor: $nested ? 'none' : theme.colors.accent,
+      background: theme.colors.primary.lighter,
+      color: theme.colors.primary.contrastText,
+      fontWeight: theme.bold,
+
+      '.marker': {
+        background: theme.colors.secondary.light,
+      },
+    },
+
+    '&:hover': {
+      background: theme.colors.primary.lighter,
+    },
+    '&:focus, &:hover': {
+      color: theme.colors.primary.contrastText,
+    },
+  };
+
+  if ($nested) {
+    css.fontSize = '11px';
+    css.background = 'none';
+    css.fontWeight = theme.regular;
+    css.paddingLeft = '96px';
+    css.minHeight = '40px';
+  }
+
+  return css;
+};
+
+const SideNavExternalLink = ({ children, href, icon }) => {
+  return (
+    <StyledSideNavExternalLink href={href} target="_blank">
+      <SideNavItemIcon as={icon} />
+      {children}
+      <SideNavItemIcon
```

Comment:
This is inheriting `margin-right: 16px;` from SideNavItemIcons default props... and it's not exactly lined up with the other icons, If we want them aligned that is.

![Screenshot 2022-12-28 at 1 55 03 PM](https://user-images.githubusercontent.com/11967646/209870973-9929d0c5-a88f-44a3-b555-faf378f3e6c9.png)